pykeeb/keyswitch_mount.py

```diff
@@ -17,6 +17,15 @@ def __init__(self, transformations, ik=False, switch_type='alps', mount_length=D
 		self.mount_width = mount_width
 		self.switch_type = switch_type
 		mx_hole = Cube([mx_width, mx_length, self.thickness], center=True)
+
```

Comment:
Looks great, but editing to make them optional.  Might not work nice with FDM printers (I wouldn't know).